cinema/serializers.py

```diff
@@ -1 +1,106 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Actor,
+    Genre,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = (
+            "id", "first_name",
+            "last_name", "full_name",
+        )
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = (
+            "id", "name",
+        )
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = (
+            "id", "name",
+            "rows", "seats_in_row",
+            "capacity",
+        )
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = (
+            "id", "title",
+            "description", "duration",
+            "genres", "actors",
+        )
+
+
+class MovieListSerializer(MovieSerializer):
+    genres = serializers.SlugRelatedField(
+        many=True,
+        read_only=True,
+        slug_field="name",
+    )
+    actors = serializers.SlugRelatedField(
+        many=True,
+        read_only=True,
+        slug_field="full_name",
+    )
+
+    class Meta:
+        model = Movie
```

Comment:
Your MovieListSerializer inherits from MovieSerializer. Do you really need class Meta here with exact same fields ?